README.md

```diff
@@ -81,6 +81,8 @@ The most recent version of the User Guide (RAJA develop branch): https://raja.re
 
 To access docs for other RAJA released versions: https://readthedocs.org/projects/raja/
 
+We maintain a repository for the RAJA portaility suite tutorial material: https://github.com/LLNL/raja-suite-tutorial
```

Comment:
Does this actually add a link to the github project in the docs?  I haven't tried this way, but I believe you need to follow the syntax you used in your changes to the docs/index.rst file.  You may want to activate your branch on RTD to be sure.

Reply:
Yes it does add the a clickable link (I tried it out in the preview mode). It is also consistent with the other links above.

Reply:
Good to know. So I guess the other format is for when you want the link to have a specific label.